Keas.Mvc/Startup.cs

```diff
@@ -42,7 +42,7 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<KfsApiSettings>(Configuration.GetSection("KfsApi"));
 
             // setup services
-            services.AddSingleton<IIdentityService, IdentityService>();
+            services.AddScoped<IIdentityService, IdentityService>();

```

Comment:
ah maybe it doesn't like it because you injected the db context?  If so then it's ok, or you could always pass along the context to the method itself too.  But scoped is fine if that's the case, it's clean

Reply:
Yeah....I got a scoped in Singleton error initially, hence the change. It felt ok, but I can swap if that's better.